receiver/windowsperfcountersreceiver/testdata/config-nocounters.yaml

```diff
@@ -1,5 +1,10 @@
 receivers:
   windowsperfcounters:
+    metrics:
```

Comment:
Can we have cases to validate not having metrics? Maybe one without a metrics block at all, and another where a counter doesn't reference any metric in the metrics block?

Reply:
Added